frontend/src/pages/profile/components/user-activity/styles.module.scss

```diff
@@ -15,7 +16,7 @@
 
 .activityWrapper {
   display: flex;
-  max-width: 710px;
+  width: 710px;
```

Comment:
Not sure if we need to lock sections width directly

Reply:
With max-width, the scroll doesn't appear in the firefox and table collapses

Reply:
Okay, let's take a look